front/pages/api/poke/workspaces/[wId]/data_sources/[dsId]/search.ts

```diff
@@ -52,7 +53,32 @@ async function handler(
 
   switch (req.method) {
     case "GET": {
-      return handleDataSourceSearch({ req, res, dataSource });
+      // I could not find a way to make the query params be an array if there is only one tag.
```

Comment:
Why not using zod to handle this parsing?

Reply:
Let's not leak zod all over the place until we are commited to using it in front. I'm still very bearish on adding zod without an explicit commitment to remove iots, the risk of having a mixed bag of libraries to do the same thing is very high.

Reply:
@flvndvd for the reason @spolu mentioned : i didn't want to start adding zod directly in front (or anywhere else).

Reply:
@flvndvd but I'm happy to switch entierly from io-ts to zod in another PR if we want to.